<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
    <line>
      <![CDATA[<img src='D:\Test_Automation\TestNgDemo\.\facebook.png' height='100' width='100'/> ]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="1597" started-at="2018-05-20T12:04:56Z" finished-at="2018-05-20T12:04:57Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="1597" started-at="2018-05-20T12:04:56Z" finished-at="2018-05-20T12:04:57Z">
      <class name="com.testng.demo.NewTest">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:com.testng.demo.NewTest@3abbfa04]" name="beforeSuite" is-config="true" duration-ms="11650" started-at="2018-05-20T17:34:44Z" finished-at="2018-05-20T17:34:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.testng.demo.NewTest@3abbfa04]" name="beforeTest" is-config="true" duration-ms="1" started-at="2018-05-20T17:34:56Z" finished-at="2018-05-20T17:34:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="f()[pri:0, instance:com.testng.demo.NewTest@3abbfa04]" name="f" duration-ms="24" started-at="2018-05-20T17:34:56Z" finished-at="2018-05-20T17:34:56Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [FaceBo] but found [Facebook – log in or sign up]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [FaceBo] but found [Facebook – log in or sign up]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.testng.demo.NewTest.f(NewTest.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[<img src='D:\Test_Automation\TestNgDemo\.\facebook.png' height='100' width='100'/> ]]>
            </line>
          </reporter-output>
        </test-method> <!-- f -->
        <test-method status="PASS" signature="f2()[pri:1, instance:com.testng.demo.NewTest@3abbfa04]" name="f2" duration-ms="828" started-at="2018-05-20T17:34:56Z" finished-at="2018-05-20T17:34:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f2 -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:com.testng.demo.NewTest@3abbfa04]" name="afterTest" is-config="true" duration-ms="1" started-at="2018-05-20T17:34:57Z" finished-at="2018-05-20T17:34:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:com.testng.demo.NewTest@3abbfa04]" name="afterSuite" is-config="true" duration-ms="1" started-at="2018-05-20T17:34:57Z" finished-at="2018-05-20T17:34:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- com.testng.demo.NewTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
